.kuzzlerc

```diff
@@ -116,14 +165,60 @@
           }
         },
         "%kuzzle": {
```

Comment:
ok... hapilly I have written the rules with the team ^^ definitely need to properly document all this